build.gradle.kts

```diff
@@ -69,6 +70,8 @@ dependencies {
     implementation("org.zalando:problem-spring-web-starter:$problemSpringWebStartVersion")
 
     testImplementation("io.mockk:mockk:$mockkVersion")
+    testImplementation("org.mockito:mockito-junit-jupiter:$mockitoVersion")
```

Comment:
Vi har allerede et mock-rammeverk tilpasset Kotlin: https://mockk.io/

Kan jo vurdere det :)

Reply:
Har vel også "fjernet" Mockito fra Spring-boot-starter-test:

`    testImplementation("org.springframework.boot:spring-boot-starter-test") {
        exclude(group = "org.junit.vintage")
        exclude(group = "org.mockito")
    }`

Reply:
Jeg ville brukt Mockk hvis jeg skulle lagd en ren mock-test. Men iom at jeg lagde en SpringBoot-test, så brukte jeg `org.springframework.boot.test.mock.mockito.MockBean` for å integrere mockede bønner inn i ApplicationContexten. Og den støtter Mockito, så derfor valgte jeg det. Jeg skal innrømme at jeg ikke gjorde noen mer omfattende vurdering av det enn det. Jeg kunne droppet MockBean og heller skrevet litt mer verbose kode ala dette tror jeg:
```
   @TestConfiguration
    class ControllerTestConfig {
        @Bean
        fun unleash() = mockk<Unleash>()
    }
```
Så hvis det er sterke følelser for mockk og mot Mockito, så kan jeg godt gjøre det. :)

Reply:
Jeg har kanskje ikke veldig sterke følelser, men ser at det finnes støtte for dette via https://github.com/Ninja-Squad/springmockk. 

Se også https://spring.io/guides/tutorials/spring-boot-kotlin/ for eksempelkode.

Jeg tror i hvert fall ikke at det skulle være nødvendig å ha to ulike mocklibs i koden hvis vi kan unngå det. Mockk er jo skreddersydd for Kotlin i tillegg :)

Reply:
Jeg tar poenget, og endrer. Takk for referanser til tutorial/bibliotek. :)